libs/internal/tests/data_model_serialization_test.cpp

```diff
@@ -27,7 +29,8 @@ TEST(SDKDataSetTests, DeserializesZeroSegments) {
         boost::json::value_to<tl::expected<data_model::SDKDataSet, JsonError>>(
```

Comment:
Some of these tests change in the (not yet issued) evaluator PR, due to slight changes in the data model.

Reply:
Yeah, I am depending a bit on testing for that. Mainly checked conditions matched manes.